src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -237,20 +373,36 @@ export default class BrowserCell extends Component {
       content = <span>&nbsp;</span>;
       classes.push(styles.empty);
     } else if (type === 'Pointer') {
+      const defaultPointerKey = await localStorage.getItem(value.className) || 'objectId';
+
+      let dataValue = value.id;
+      if( defaultPointerKey !== 'objectId' ) {
+        dataValue = value.get(defaultPointerKey);
+        if ( dataValue && typeof dataValue === 'object' ){
+          if ( dataValue instanceof Date ) {
```

Comment:
This looks like strange formatting with the space inside the brackets. Do we run lint over this usually?